src/landing/community.md

```diff
@@ -72,7 +71,7 @@ Writing a new plugin is often easier than patching sbt's core code.
 Check out the list of [community plugins][Community-Plugins].
 
 plugin and sbt core development is discussed on the [sbt-dev list][ml] and
-on [Gitter sbt/sbt-dev][gitter2].
+on [Gitter sbt/sbt][gitter].
```

Comment:
Could you link to https://github.com/sbt/sbt/discussions instead please?

Reply:
done!